pkg/metrics.go

```diff
@@ -0,0 +1,86 @@
+package ripley
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	_ "net/http/pprof"
+
+	"github.com/VictoriaMetrics/metrics"
+)
+
+var metricsRequestReceived = make(chan bool)
```

Comment:
No global variables, please. Use the same pattern we use for the channels in https://github.com/loveholidays/ripley/blob/main/pkg/client.go#L34

Reply:
The desire to use global was simply too strong to overcome.